test/OpenTelemetry.Tests/Trace/LinkTests.cs

```diff
@@ -73,18 +73,6 @@ public void Equality()
         Assert.True(link1.Equals(link3));
     }
 
-    [Fact(Skip = "ActivityLink.Equals is broken in DS7 preview: https://github.com/dotnet/runtime/issues/74026")]
-    public void Equality_WithAttributes()
-    {
-        var link1 = new Link(this.spanContext, this.tags);
-        var link2 = new Link(this.spanContext, this.tags);
-        object link3 = new Link(this.spanContext, this.tags);
-
-        Assert.Equal(link1, link2);
-        Assert.True(link1 == link2);
-        Assert.True(link1.Equals(link3));
```

Comment:
Alternatively, this test could be updated to reflect the new [documented behavior](https://github.com/dotnet/dotnet-api-docs/pull/8316/files#diff-6235b9194b43a6136628865e0ff2c872d74f572d4ba34bcfded5051af27ba92bR140)

```suggestion
        Assert.NotEqual(link1, link2);
        Assert.False(link1 == link2);
        Assert.False(link1.Equals(link3));
```